api/controllers/WebActionsController.js

```diff
@@ -332,3 +358,24 @@ class ErrorMobileCommonsJoin extends Error {
     super(message);
   }
 }
+
+/**
+ * Create initial form data for Mobile Commons join request
+ * Form data depends on if a user is opting in themselves or referring friends
+ */
+function createMobileCommonsFormData(formData) {
+  return formData.campaign && formData.friends
```

Comment:
Ah, I'd really shy away from using ternary operators for anything that's bigger than one line.

And while the logic here does work for now, it does feel super specific to knowledge of campaigns and the beta-only friend form. The two items in the conditional (`.campaign` and `.friends`) aren't used in either the true or false evaluations, so it's not entirely clear what the conditional's for without prior knowledge. So something to think about for the future could be something more descriptive like...

```
if (formData.betasOnly) {
  'person[phone]': formData.phone
} else {
  // The normal request object
}
```